README.md

```diff
@@ -285,6 +285,11 @@ ReactDOM.render(
 - `thisContext: any`: `this` context to use when calling the callback function.
 - => Returns an `off` method to cancel the event subscription.
 
+#### `onWebSocketError(callback, thisContext) => Function` - shorthand for `.on('websocket_error', ...)`
```

Comment:
Same here, no need for the `WebSocket`, let's call this `onError`.

Reply:
Done.